resources/AdvancedTabComponents/NtripClientTab.qml

```diff
@@ -125,6 +125,24 @@ Item {
                 enabled: !connected
             }
 
+            ComboBox {
+                id: outputType
+
+                editable: false
+
+                model: ListModel {
+                    ListElement {
+                        text: "RTCM"
+                    }
+
+                    ListElement {
+                        text: "SBP"
+                    }
+
+                }
+
+            }
```

Comment:
Might be better to do checkbox instead since we only allow either RTCM or SBP. Not sure if any future elements will be added.